pkg/kapp/resources/exists_options.go

```diff
@@ -0,0 +1,9 @@
+// Copyright 2021 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package resources
+
+type ExistsOpts struct {
+	SameUID bool
+}
```

Comment:
i would move this to resources.go next to Resources interface (since it's part of the interface).